### ISSUE TITLE
DynamoDB Global Table Import support

### ISSUE THREAD

**Issue by kayengar** [aws-cloudformation/cloudformation-coverage-roadmap#863]
## Cloudformation DynamoDB Global Table Import support 

Quick Sample Summary:
1. **Title** -> AWS::DynamoDB::GlobalTable Imports
2. **Scope of request** -> 
AWS::DynamoDB::GlobalTable has Cloudformation support today, but an existing GlobalTable cannot be imported into a Cloudformation stack. We need to be able to import existing global tables into Cloudformation stacks. 
3. **Expected behavior** -> 
- When we import a CFN stack with DynamoDB global tables, it should be able to recognize and import the GlobalTable as a new resource into the template. 
- (Additional feature) - If it can determine 2017 Global Tables and can import it into a new stack, it is an added advantage since there is no way currently to convert 2017 Global Table to 2019 Global Table outside of console. 
5. **Links to existing API doc** ->
 - https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-dynamodb-globaltable.html
 - https://github.com/aws-cloudformation/cloudformation-coverage-roadmap/issues/57#issuecomment-844337982
6. Category tag (optional) -> DB

**Comment by mrinaudo-aws** [comment]
Hi @kayengar !  I just tested by creating an Amazon DynamoDB global table with the AWS Management Console.  I was, subsequently, able to import the global table with a template describing the table as an `AWS::DynamoDB::GlobalTable` resource type.  Are you still not able to perform the import?  If so, can you please provide more information?  Thank you!

**Comment by kayengar** [comment]
@mrinaudo-aws Thanks for getting back to me. Let me try it. Also, would we be able to import a 2017 Global Table into a new CFN stack?

**Comment by benkehoe** [comment]
Note that `::GlobalTable` is not listed as supported in the documentation. https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/resource-import-supported-resources.html

**Comment by mrinaudo-aws** [comment]
Hi @benkehoe !  I will convey your feedback to the team, thank you!

**Comment by mrinaudo-aws** [comment]
Hi @benkehoe !  The `AWS::DynamoDB::GlobalTable` resource is now listed in the documentation [page](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/resource-import-supported-resources.html) you mentioned.  Thank you again for your feedback!

**Comment by mrinaudo-aws** [comment]
Hi @kayengar !  I am closing this issue, please feel free to reopen it if you have issues with performing the import.  Thank you!

**Comment by animaxcg** [comment]
This is still an issue

```
aws --version
aws-cli/2.9.5 Python/3.9.11 Darwin/21.6.0 exe/x86_64 prompt/off
    aws cloudformation create-change-set \
    --stack-name ${stackName} --change-set-name ImportChangeSet \
    --change-set-type IMPORT \
    --resources-to-import "[{\"ResourceType\":\"AWS::DynamoDB::GlobalTable\",\"LogicalResourceId\":\"${resource}\",\"ResourceIdentifier\":{\"TableName\":\"${tableName}\"}}]" \
    --template-url https://${bucket}.s3.${region}.amazonaws.com/${s3Prefix}/cfn/${template}--capabilities CAPABILITY_NAMED_IAM  --parameters "${params}"
  

An error occurred (ValidationError) when calling the CreateChangeSet operation: ResourceTypes [AWS::DynamoDB::GlobalTable] are not supported for Import

```

**Comment by kayengar** [comment]
Yes, this is still an issue. Sorry I have not worked on this for a while.

**Comment by kddejong** [comment]
@kayengar I have been able to do this import.  Can you retest your situation and see if its working.
